ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -918,6 +918,24 @@ AHU_DFSS_CHWDT_DTM_RTC_RWISOVPC_OAFC_FDPSM2X:
   - OAFC
   - FDPSM2X
 
+AHU_SFSS_SFVSC_EFSS_EFVSC_HTWHLZTC_HWZTC_CHWZTC_SSPC_RSPC_CO2DSPMADC_MTM_FDPM3X:
```

Comment:
I would expect this to have some form of economizer control (ECONZ, perhaps). You have nothing to model the outside air damper right now.